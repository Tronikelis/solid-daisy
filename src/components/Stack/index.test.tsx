import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Stack } from ".";

describe("<Stack />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Stack>foo</Stack>);
        expect(container).toMatchSnapshot();
    });
});
