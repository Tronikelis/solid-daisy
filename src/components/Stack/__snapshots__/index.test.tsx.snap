// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Stack /> > renders correctly 1`] = `
<div>
  <div
    class="flex flex-col gap-4"
  >
    foo
  </div>
</div>
`;
